content/STRUCTURE.de.json

```diff
@@ -0,0 +1,369 @@
+[
+    {"path": "/",
+     "title": "",
+     "menu": ["Startseite"],
+     "generator": "html",
+     "meta": {"input": "index.html"}},
+
+    {"path": "/incus/",
+     "generator": "alias",
+     "meta": {"target": "/incus/introduction/"}},
+
+    {"path": "/incus/introduction/",
+     "title": "Incus - Einführung",
+     "menu": ["Incus", "Einführung"],
+     "generator": "markdown",
+     "meta": {"input": "incus/introduction.de.md"}},
```

Comment:
You're referencing a bunch of `.de.md` files in this PR which don't exist yet, this will cause issues building the site or result in 404 for all those pages.

The reason this isn't failing right now during the build is that you didn't add German to `content/CONFIG.json`. Once you do, the website will actually start building the translate content and will require all referenced `.de.md` files to exist.